pkg/riff/commands/handler_create.go

```diff
@@ -130,9 +132,13 @@ func NewHandlerCreateCommand(c *cli.Config) *cobra.Command {
 	opts := &HandlerCreateOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "create",
-		Short:   "<todo>",
-		Example: "<todo>",
+		Use:   "create",
+		Short: "deploy an application, function or image to handle http requests",
```

Comment:
I think this should mention the handler since that is the thing being created. Probably deserves a long description too.